bin/setup-services.sh

```diff
@@ -10,6 +10,14 @@ set -eo pipefail
 python bin/s3_cli.py delete "${UPLOAD_DEFAULT_URL}"
 python bin/s3_cli.py create "${UPLOAD_DEFAULT_URL}"
 
+# Set up GCS
+# FIXME bug 1827506: conditionally set UPLOAD_DEFAULT_URL TO GCS URL
+# (i.e. https://gcs-emulator:4443/storage/v1/b/{bucket_name})
+# when CLOUD_SERVICE_PROVIDER == "GCP" and pass that in here instead
+# of publicbucket.
+python bin/gcs_cli.py delete publicbucket
+python bin/gcs_cli.py create publicbucket
```

Comment:
Currently, I'm constructing the `endpoint_url` within `gcs_cli.py` and only accepting a bucket name here instead of either a bucket name or a URL.

Passing in a URL here is a little confusing to me, though it looks like for the S3 CLI, a URL is passed, and this URL is used throughout the source code. I thought I could leave this as-is for now, and perhaps this can be revisited once the [storage API is refactored](https://bugzilla.mozilla.org/show_bug.cgi?id=1564452), and once we [add GCS](https://bugzilla.mozilla.org/show_bug.cgi?id=1827506) into the source code. What do you think? I can make the code comment less prescriptive.

Reply:
@smarnach ^^^ What do you prefer?

Reply:
I cleaned up the comment to be less prescriptive in the meantime. Other than this detail, the patch is ready for review, so I've marked it as ready!

Reply:
I don't see an advantage to using a URL here, so let's keep it the way it is. As Bianca already said, we can always change it later if we need to.